### ISSUE TITLE
Clicking on update in popup replaces current tab content

### ISSUE THREAD

**Issue by freaktechnik** [sneakypete81/updatescanner#286]
* OS version: elementary OS
 * Firefox version: 64.0a1
 * Update Scanner version: 4.3.0
 * Firefox language: en-US

#### Steps to reproduce the problem:
- Have an update to some site in the scanner
- Open popup
- Click on update

#### What you expected to happen:
A new tab with the diff opens

#### What actually happened:
The diff is loaded in the current tab, even app tabs etc.

**Comment by sneakypete81** [comment]
This is the expected behaviour - it works the same as the other popups (history, bookmarks, etc).
Maybe allowing middle-click (etc.) to open in a new tab would be useful?

If you click "Show all updates" the pages load in new tabs, so that might be a good workaround for you.

**Comment by freaktechnik** [comment]
Is it for app tabs too? Because at least there the library items open a new tab and don't replace the app tab contents. Whereas the extension doesn't differentiate.

**Comment by sneakypete81** [comment]
Interesting, I hadn't noticed that the library popups behave differently when a pinned tab is open. Thanks for reporting.

**Comment by sneakypete81** [comment]
#297 has some further thoughts.

I'm leaning towards changing the click behaviour in the popup to always open a new tab, since that's probably the least surprising behaviour.

**Comment by sneakypete81** [comment]
Added to 4.3.3beta2.

**Comment by yermulnik** [comment]
@sneakypete81 Is there an option to disable this new behavior? Having several dozens of updates in one go I don't feel happy having new tab for every updated page =(

**Comment by yermulnik** [comment]
Any clue on this? Maybe a configurable option please?

**Comment by sneakypete81** [comment]
It sounds like you're clicking each update one at a time, correct? If clicking the "close tab" button is too awkward, CTRL-W is a useful shortcut.

I'd prefer not to add preferences for things like this, but would be useful to understand your usage. I'd expect clicking "show all updates" would be easiest if you have lots of changes to review.

**Comment by yermulnik** [comment]
I used to previous behavior when I click updatescanner notification icon and see a popup with a list of all updated pages, then clicking one by one I can see if those changes worth looking into or not worth attention and proceed to next one. Everything is done in one single tab. If I've got e.g. 20 updated pages I now need to click the link, then ctrl-w, then click next link as opposite to clicking only once. Never tried "show all updates" but I seem to think this is going to open a new tab for every updated page but I'm not really keen to have e.g. 20 new tabs open in addition to already opened e.g. 30 tabs.
Looking into #286 I don't feel this change was massively requested by lots of users but I don't want to revert it since it might be useful for some of them with e.g. quite small amount of monitored pages. Thus asking for a configurable option please.

**Comment by sneakypete81** [comment]
Ok, thanks for the detailed explanation. Please open a new Issue to track this.

In the meantime, I'd suggest either using the sidebar (fewer clicks but harder to locate changed pages) or open a new Firefox window (CTRL-N) and click "Show All Changes" there (fewer clicks, changes kept separate from your existing tabs).

**Comment by yermulnik** [comment]
@sneakypete81 Thanks. Done: #395
